dde-file-manager-lib/dialogs/computerpropertydialog.cpp

```diff
@@ -100,7 +103,7 @@ void ComputerPropertyDialog::initUI()
 
         gridLayout->addWidget(keyLabel, row, 0, Qt::AlignRight | Qt::AlignTop);
         gridLayout->addWidget(valLabel, row, 1, Qt::AlignLeft | Qt::AlignTop);
-        gridLayout->setRowMinimumHeight(row, valLabel->heightForWidth(gridLayout->columnMinimumWidth(1)));
+        gridLayout->setRowMinimumHeight(row, valLabel->heightForWidth(valLabel->width())-10);
```

Comment:
Is there any reason to use the number `10` or just because this magic number just works?

Reply:
Just a magic number.

Reply:
Then I cannot accept this since it may still have problem in some other unexpected cases. Sorry.

Reply:
@BLumia Then what about the problem in the current dde-file-manager?

Reply:
@BLumia The reason why I used the number 10 is because without substracting by that number we get some extra spaces.
Without substracting by 10:
![DeepinScreenshot_dde-desktop_20190618212637](https://user-images.githubusercontent.com/33563064/59697533-cb086300-920f-11e9-8502-e1d4ff5c3ef5.png)
After substracting by 10:
![DeepinScreenshot_dde-desktop_20190618213030](https://user-images.githubusercontent.com/33563064/59697822-584bb780-9210-11e9-8404-3152bcedc11e.png)

Reply:
I still don't think using a magic number is a good idea since this job should be done by using layout correctly. Maybe you can use [GammaRay](https://github.com/KDAB/GammaRay) or something else to debug the layout issue :)